documentation/src/main/asciidoc/topics/proc_generating_protostream_sci.adoc

```diff
@@ -1,30 +1,26 @@
 [id='generating_protostream_sci-{context}']
 = Generating Serialization Context Initializers
-A ProtoStream `SerializationContext` contains Protobuf type definitions for
-custom Java objects, loaded from Protobuf schemas, and the accompanying
-marshallers for those objects.
+Annotate Java objects in your classes to include them in an implementation of the ProtoStream `SerializationContextInitializer` interface so that the ProtoStream library can encode your Java objects to Protobuf format and {brandname} can transmit and store them.
```

Comment:
I think this needs rewording. Annotating Java classes is required so that ProtoStream knows what fields to include in the `.proto` definition and how to generate the marshaller implementations. Including a class in a `SerializationContextInitializer` definition is achieved by adding the `SomeObject.class` to the `includeClasses = {} ` field.

I would also avoid the "can transmit and store them" and just say that it's so that the objects can be encoded in the Protobuf format. I believe we already explain why marshalling is required at the begining of the chapter.